src/custom_minimizer.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Feb  5 13:00:53 2016
+
+@author: SamWitte
+
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from scipy.optimize import minimize
+
+
+"""
+The general idea here is this:
+Can I create a minimization procedure that is more efficient than that standard basinhopping (and finds the actual minimium).
```

Comment:
Lines too long. I agree Pep8's rule of 80 is too tight, but 90(ish) is more doable. Either way, try to not go over 100. And anything over 120 doesn't even fit in git's window.